packages/web-app-files/src/components/SideBar/TagsPanel.vue

```diff
@@ -181,7 +182,7 @@ export default defineComponent({
       createOption,
       isOptionSelectable,
       showSelectNewLabel,
-      save
+      save,
```

Comment:
Please always run `pnpm lint --fix` on save in your IDE or before you commit. ;-) This comma fails in the linting step.